codec-http/src/test/java/io/netty/handler/codec/http/HttpContentCompressorTest.java

```diff
@@ -603,6 +603,57 @@ public void test100Continue() throws Exception {
         assertThat(ch.readOutbound(), is(nullValue()));
     }
 
+    @Test
+    public void test1xxInformationalResponse() throws Exception {
+        FullHttpRequest request = newRequest();
+        HttpUtil.set100ContinueExpected(request, true);
+
+        EmbeddedChannel ch = new EmbeddedChannel(new HttpContentCompressor());
+        ch.writeInbound(request);
+
+        FullHttpResponse continueResponse = new DefaultFullHttpResponse(
+            HttpVersion.HTTP_1_1, HttpResponseStatus.CONTINUE, Unpooled.EMPTY_BUFFER);
+        ch.writeOutbound(continueResponse);
```

Comment:
How about we also have a test where we skip the CONTINUE response and immediately send the early hints?
I understand their main use case are to be the first response that goes out.

Reply:
You are right, I was testing both early hints and multiple 1xx responses in the same test.
I'll split the test in two